packages/dx-react-chart-demos/src/demo-sources/chart-featured-grid-integration/bootstrap4/chart-featured-grid-integration.jsx

```diff
@@ -0,0 +1,214 @@
+import * as React from 'react';
+import { Card } from 'reactstrap';
+import {
+  RowDetailState,
+  DataTypeProvider,
+} from '@devexpress/dx-react-grid';
+import {
+  Stack,
+} from '@devexpress/dx-react-chart';
+import {
+  Chart,
+  BarSeries,
+  ArgumentAxis,
+  ValueAxis,
+  Legend,
+  Grid as ChartGrid,
+} from '@devexpress/dx-react-chart-bootstrap4';
+import {
+  Grid, Table, TableBandHeader, TableHeaderRow,
+  TableRowDetail,
+} from '@devexpress/dx-react-grid-bootstrap4';
+import { sales as dataBase, citiescount as cities, dataforregions as dataforgrid } from '../../../demo-data/data-for-grid';
+
+const nullComponent = () => null;
+const START_YEAR = 2013;
+const END_YEAR = 2016;
+const AxisLabelComponent = ({
+  text,
+  ...restProps
+}) => <ValueAxis.Label text={`$ ${text}`} {...restProps} />;
+
+const CurrencyFormatter = ({ value }) => (
+  <div>
+    {'$ '}
+    {value}
+  </div>
```

Comment:
You can use [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in this case:

```
<div>{`$ ${value}`}</div>
```

Reply:
The div element can be removed too. Just write 
```
const Formatter = `$${value}`
```